app/build.gradle

```diff
@@ -12,7 +12,7 @@ android {
         minSdkVersion 21
         targetSdkVersion 29
         versionCode 11
-        versionName "1.9.0"
+        versionName "2.0.0"
```

Comment:
Let's go with 1.10.0 instead of 2.0.0. But perhaps better to just leave the version for now at 1.9.0, and then just before I make a new release (which might include other PRs) I will bump the version number.

Reply:
Agreed, will remove version bump from this PR.